Upstream release contains config.{guess,sub} from six years ago that
don't handle Apple silicon. This patches minimally around it.

--- a/config.sub	2018-02-22 00:00:00
+++ b/config.sub	2024-07-23 21:45:09
@@ -450,6 +450,9 @@
 	| ymp-* \
 	| z8k-* | z80-*)
 		;;
+	arm64-apple)
+		basic_machine=aarch64-apple
+		;;
 	# Recognize the basic CPU types without company name, with glob match.
 	xtensa*)
 		basic_machine=$basic_machine-unknown
